addons/controllers/testdata/test-antrea-calico-webhook-manifests.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
```

Comment:
Generic service name might be mixed up with other webhooks. Better to give it a name related to Antrea and Calico.

Reply:
As of now, the design is for a common webhook server and webhook service that allso share the same certificates. Hence the generic name